README.md

```diff
@@ -12,94 +12,95 @@
+integrates QNNs with [PyTorch](https://pytorch.org). 
+Thanks to the gradient algorithms in Qiskit Machine Learning, this includes automatic differentiation. 
+The overall gradients computed by PyTorch during the backpropagation take into account quantum neural 
+networks, too. The flexible design also allows the building of connectors to other packages in the future.
+
+## Installation and documentation
+
+We encourage installing Qiskit Machine Learning via the `pip` tool, a `Python` package manager.
 
 ```bash
 pip install qiskit-machine-learning
 ```
 
-**pip** will handle all dependencies automatically and you will always install the latest
-(and well-tested) version.
+`pip` will install all dependencies automatically, so that you will always have the most recent
+stable version.
 
-If you want to work on the very latest work-in-progress versions, either to try features ahead of
-their official release or if you want to contribute to Machine Learning, then you can install from source.
-To do this follow the instructions in the
+If you want to work instead on the very latest _work-in-progress_ versions of Qiskit Machine Learning, 
+either to try features ahead of
+their official release or if you want to contribute to the library, then you can install from source.
+For more details on how to do so and much more, follow the instructions in the
  [documentation](https://qiskit-community.github.io/qiskit-machine-learning/getting_started.html#installation).
 
 ### Optional Installs
 
-* **PyTorch**, may be installed either using command `pip install 'qiskit-machine-learning[torch]'` to install the
+* **PyTorch** may be installed either using command `pip install 'qiskit-machine-learning[torch]'` to install the
   package or refer to PyTorch [getting started](https://pytorch.org/get-started/locally/). When PyTorch
   is installed, the `TorchConnector` facilitates its use of quantum computed networks.
 
-* **Sparse**, may be installed using command `pip install 'qiskit-machine-learning[sparse]'` to install the
-  package. Sparse being installed will enable the usage of sparse arrays/tensors.
+* **Sparse** may be installed using command `pip install 'qiskit-machine-learning[sparse]'` to install the
+  package. Sparse being installed will enable the usage of sparse arrays and tensors.
+
+* **NLOpt** is required by the global optimizers. [`NLOpt`](https://nlopt.readthedocs.io/en/latest/) 
+  can be installed with `pip install 'qiskit-machine-learning[nlopt]'`.
 
 ## Migration to Qiskit 1.x
 > [!NOTE]
-> Qiskit Machine Learning learning depends on Qiskit, which will be automatically installed as a 
+> Qiskit Machine Learning depends on Qiskit, which will be automatically installed as a 
 > dependency when you install Qiskit Machine Learning. If you have a pre-`1.0` version of Qiskit 
 > installed in your environment (however it was installed), and wish to upgrade to `1.0`, you 
```

Comment:
Maybe we should think about rephrasing this - specifically the wish to upgrade - that was optional as such before so if you didn't want to you did not have to - but now, if you wish to use the same environment then you have to upgrade to 1.0

Reply:
I rephrased it to clarify the requirement of Qiskit >=1.0.